org.gnome.Firmware.json

```
@@ -0,0 +1,72 @@
+{
+  "app-id": "org.gnome.Firmware",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
```

```suggestion
  "runtime-version": "3.34",
```
Any reason for not using 3.34?

---

No reason, but I've not tested it with that runtime version. Give me a few mins and I'll do just that.

---

Seems to work fine with the new version.